app/src/main/java/hm/orz/chaos114/android/tumekyouen/modules/kyouen/KyouenActivity.kt

```diff
@@ -194,29 +194,29 @@ class KyouenActivity : DaggerAppCompatActivity(), KyouenActivityHandlers {
         binding.fragmentContainer.addView(tumeKyouenView)
 
         oldView.animate()
-                .translationX(oldTranslationX)
-                .setDuration(250).interpolator = AccelerateInterpolator()
+            .translationX(oldTranslationX)
+            .setDuration(250).interpolator = AccelerateInterpolator()
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/noboru-i/kyouen-android/blob/190e05f3009232f6a77bdf0cc25a1773db9f7ee6/app/src/main/java/hm/orz/chaos114/android/tumekyouen/modules/kyouen/KyouenActivity.kt#L198"></span>This expression contains a magic number. Consider defining it to a well named constant.</td>
    </tr>
  </tbody>
</table>